modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -516,6 +517,56 @@ public void testUpdateObjectRelationship() throws Exception {
 			objectRelationship6.getObjectFieldId2());
 
 		Assert.assertFalse(objectField2.isRequired());
+
+		ObjectRelationship systemObjectRelationship =
+			_objectRelationshipLocalService.addObjectRelationship(
+				TestPropsValues.getUserId(),
+				_objectDefinition1.getObjectDefinitionId(),
+				_objectDefinition2.getObjectDefinitionId(), 0,
+				ObjectRelationshipConstants.DELETION_TYPE_PREVENT,
+				LocalizedMapUtil.getLocalizedMap("Able"), StringUtil.randomId(),
+				true, ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
+
+		Assert.assertEquals(
+			LocalizedMapUtil.getLocalizedMap("Able"),
+			systemObjectRelationship.getLabelMap());
+
+		systemObjectRelationship =
+			_objectRelationshipLocalService.updateObjectRelationship(
+				systemObjectRelationship.getObjectRelationshipId(), 0,
+				ObjectRelationshipConstants.DELETION_TYPE_DISASSOCIATE, false,
+				LocalizedMapUtil.getLocalizedMap("Able"));
+
+		Assert.assertEquals(
+			ObjectRelationshipConstants.DELETION_TYPE_DISASSOCIATE,
+			systemObjectRelationship.getDeletionType());
+
+		// Changes from outside allowed bundles can only edit label
```

Comment:
Nice comment :smile: 
What do you think about this change:
**Requests** from **not-allowed** bundles can only **update the** label

IDK, I'm not sure about the term "outside allowed bundles"